pkg/apis/iam/v0alpha1/types_servier_account.go

```diff
@@ -30,15 +30,15 @@ type ServiceAccountList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
 
-	Items []ServiceAccount `json:"items,omitempty"`
+	Items []ServiceAccount `json:"items"`
```

Comment:
in v33, they start complaining about having omitempty on Items.  This change will avoid all these warnings (and presumably be more accurate)

https://github.com/grafana/grafana/pull/105307/files#diff-1b75b852631e8787c9fae2fb58c07e5afef619ce7433e11a9bada23c098d8417